module-types/Intel/FXX1600PCRPS.yaml

```diff
@@ -0,0 +1,8 @@
+---
+manufacturer: Intel
+model: FXX1600PCRPS
+power_ports:
+  - name: PSU
```

Comment:
Please change this to `- name: PSU{module}`
What happens here is that `{module}` in the name, when added to a module-bay will use the `position` definition in place of this. For instance, if the module-bay defined the position as `position: 1`, then when attached this will get called `PSU1`